@types/state-machine.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for state machine 0.2.2
+// Project: https://github.com/brophdawg11/state-machine#readme
+// TypeScript Version: 3.7.4
+
+export default StateMachine
+
+declare class StateMachine {
+    constructor(machine: MachineDefinition, initialState?: string);
+    currentState(): string;
```

Comment:
Should this be declared without the parens because it's a getter prop?  It would be accessed as `stateMachine.currentState` instead of `stateMachine.currentState()`